torchgeo/trainers/classification.py

```diff
@@ -266,147 +262,3 @@ def predict_step(
         x = batch['image']
         y_hat: Tensor = self(x).softmax(dim=-1)
         return y_hat
-
-
-class MultiLabelClassificationTask(ClassificationTask):
```

Comment:
The _right_ thing to do would be to deprecate this first and remove it in 0.7.0. Not sure how widely used it is. Deprecation is kind of annoying because you need to change all tests to acknowledge the warning message.